concepts/05 Widgets/Scheduler/040 Resources/030 Group Appointments by Resources.md

```diff
@@ -67,6 +67,88 @@ To group appointments by resources, assign an array to the [groups](/api-referen
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="appointments"
+            :groups="groups" <!-- Groups appointments by rooms and by teachers -->
+        >
+            <DxResource
+                field-expr="roomId"
+                :data-source="roomResources"/>
+            <DxResource
+                field-expr="teacherId"
+                :data-source="teacherResources"/>
+        </DxScheduler>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler, { DxResource } from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler,
+            DxResource
+        },
+        data() {
+            return {
+                groups: ['roomId', 'teacherId'],
+                appointments: [{
+                        roomId: 1,
```

Comment:
Each next nesting level should be indented by 4 spaces, not by 8 as on this line.